package.json

```diff
@@ -29,7 +29,7 @@
     "pretest": "npm run eslint",
     "test": "npm run unit",
     "prepublish": "npm run test && npm run build",
-    "storybook": "start-storybook"
+    "storybook": "start-storybook  -c ./node_modules/@exemplar/storybook-react"
```

Comment:
You can get around having to do this by creating the file `./.storybook/presets.js` and putting this in it:

```js
module.exports = [
  '@exemplar/storybook-react'
];
```

Then this can still just be `start-storybook`